ITKImageProcessingFilters/itkDream3DImage.h

```diff
@@ -0,0 +1,266 @@
+/*=========================================================================
+*
+*  Copyright Insight Software Consortium
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*         http://www.apache.org/licenses/LICENSE-2.0.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*=========================================================================*/
+#ifndef itkDream3DImage_h
+#define itkDream3DImage_h
+
+#include "itkImportDream3DImageContainer.h"
+#include "itkImage.h"
+#include "itkImageRegion.h"
+#include "itkDefaultPixelAccessor.h"
+#include "itkDefaultPixelAccessorFunctor.h"
+#include "itkPoint.h"
+#include "itkFixedArray.h"
+#include "itkWeakPointer.h"
+#include "itkNeighborhoodAccessorFunctor.h"
+
+namespace itk
+{
+  /** \class Dream3DImage
+  *  \brief Templated n-dimensional Dream3DImage class.
+  *
```

Comment:
Please add a note here about how this class uses malloc / free and explain why it is needed.